### ISSUE TITLE
Terms of Use, Search & Privacy Policy pages language setting immediately resets to English

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1199]
**TL;DR:** Change the language on the Terms of Use or Privacy Policy page to Farsi. Note that the page refreshes immediately and the language is reset to English.

This isn't Farsi-specific. If you choose Chinese or German the same behaviour exists. (I think probably it will happen for any language besides English.)

Re Terms of Use: On Production, if you choose Farsi, the footer is translated. (We don't translate legal pages, I think, so @chrisgarrity & I weren't expecting the body to be translated, but the footer should still be translatable.)

I saw this on an iPad + Safari, and Mac OS El Capitan + Chrome 56, and Ubuntu 16.04 + Firefox 51.0.1.

**Comment by jwzimmer** [comment]
I noticed that if you search for a term with the language set to Cymraeg/ Welsh, the results are set to English. If you try to change the language from the Search page to Welsh, then it resets to English immediately.

**Comment by jwzimmer** [comment]
@thisandagain said this is the same as https://github.com/LLK/scratch-www/issues/1021, so closing in favor of https://github.com/LLK/scratch-www/issues/1021.

**Comment by jwzimmer** [comment]
Reopening for now just in case this is not the same as #1021.

**Comment by chrisgarrity** [comment]
I think this is related to the Transifex transition and happens when there's a problem with importing the json translations. 
I just found an issue with unescaped quotes in l10n files (https://github.com/LLK/scratch-www/issues/1238) This caused the building the faq intl file to fail when it reached Korean. If you go to the FAQ page and select a language before Korean on the list - e.g. German, it switches fine. If you select Korean or anything after it, it immediately switches back to English. So build-locales needs to handle JSON.parse errors better.
